benchmark/_dummy-benchmarker.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Yeah it's not really a noop since it does connects to the server (a real `noop-benchmarker` would just skip the `http.get` call and write to stdout straight away). I think `_test-double-benchmarker` is better :D